doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -342,54 +348,56 @@ Follow these instructions to build the RT kernel.
    .. code-block:: none
 
       $ sudo mount /dev/sda1 /mnt
-      $ sudo cp bzImage /mnt/EFI/
+      $ sudo cp arch/x86/boot/bzImage /mnt/EFI/
       $ sudo umount /mnt
       $ sudo mount /dev/sda2 /mnt
-      $ sudo cp kernel.tar.gz -P /mnt/usr/lib/modules/ && cd /mnt/usr/lib/modules/
-      $ sudo tar zxvf kernel.tar.gz
-      $ sudo cd ~ && umount /mnt && sync
+      $ sudo cp linux-4.19.72-rt25-x86.tar.gz -P /mnt/usr/lib/modules/ && cd /mnt/usr/lib/modules/
+      $ sudo tar zxvf linux-4.19.72-rt25-x86.tar.gz
+      $ sudo cd ~ && sudo umount /mnt && sync
 
 Launch the RTVM
 ***************
 
 Update the Grub file
 ====================
 
-#. Update the ``/etc/grub.d/40_custom`` file as shown below.
+#. Mount the RTVM rootfs (Ubuntu User VM installed on the SATA disk):
+
+   .. code-block:: none
+
+      $ sudo mount /dev/sda2 /mnt
```

Comment:
please remove 
sudo mount /dev/sda2 /mnt
and keep `/etc/grub.d/40_custom` and `/etc/default/grub`